aws-datastore/src/main/java/com/amplifyframework/datastore/syncengine/Merger.java

```diff
@@ -66,6 +67,18 @@
      * @return A completable operation to merge the model
      */
     <T extends Model> Completable merge(ModelWithMetadata<T> modelWithMetadata) {
+        return merge(modelWithMetadata, value -> { });
```

Comment:
There is a `NoOpConsumer.create()`, I think? Which might be a little more explicit about the intention.

Reply:
I thought we had something like that. I was looking for `EmptyConsumer` 🤦